data_steward/cdr_cleaner/cleaning_rules/clean_survey_conduct_recurring_surveys.py

```diff
@@ -0,0 +1,176 @@
+"""
+Inserts data representing the version of a survey into survey_conduct table for surveys with custom ids. Ex COPE/Minute
+
+Recurring versions of surveys such as COPE had all of their questions and answers mapped to a single Module in the PPI
+vocabulary. The same was the case for the Minute surveys. This will make it seem like there is only one version of each
+of these surveys. Researchers need to know the version of the survey the participant took. This CR will use
+`cope_survey_semantic_version_map` which is provided to curation from the rdr team, to update the survey version in
+survey_source_concept_id and survey_source_value for each observation.
+
+There are multiple cleaning rules that clean the survey conduct table.
+This CR will clean survey_concept_id and survey_source_value for the surveys with custom concept ids.
+Another cleans survey_source_concept_id for all valid records.
```

Comment:
@hiro-mishima That is a CR that I almost have the PR out for [DC-3013](https://github.com/all-of-us/curation/pull/1500). This PR blocks its completion, but it will be ready for review soon.